### PR TITLE
Feature/slm 12

### PR DESCRIPTION
**Pull request by yossef-khaled**
**For users docs less than our items per page (less than 12 users)**
Screenshot 2022-03-05 at 23-33-33 Salam سلام
---------------------------------------------------------
**For users docs more than our items per page (3 users with 1 item per page)**
Screenshot 2022-03-05 at 23-33-53 Salam سلام

Screenshot 2022-03-05 at 23-34-07 Salam سلام

Screenshot 2022-03-05 at 23-34-16 Salam سلام

### CODE REVIEW

**Comment by yossef-khaled** [comment]
NOTE: notice the next and previous buttons.

**Comment by yossef-khaled** [comment]
**Mobile view**
Screenshot 2022-03-05 at 23-40-09 Salam سلام

File: src/Utilities/Phrases.js [hunk 1]
```diff
@@ -16,7 +16,8 @@ export const englishPhrases = {
   Vision: "Vision",
   VisionPhrase: "Becoming a hub for safe, trusted and beneficial volunteering opportunities. Encouraging people to give back along the way.",
   Story: "Our Story",
-  StoryPhrase:"Watch our story in this brief video to understand more about Salam."
+  StoryPhrase:"Watch our story in this brief video to understand more about Salam.",
+  NoUsersFromBackEnd: "There is no users"
```
Comment (MohamedAssaf) [inline src/Utilities/Phrases.js]: There are no users

**Comment by MohamedAssaf** [comment]
Great job on the next and previous buttons! Love the color. Footer looks a bit off though?

File: src/Resources/Data/Users.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import { db } from '../../Utilities/Firebase';
+
+let firstDoc = null;
+let lastDoc = null;
+
+export const getChunkOfUsers = async (nextOrPrev, itemsPerPage, setIndex) => {
+    
+    //NOTE: endAt takes the doc itself, not its index.
```
Comment (MohamedAssaf) [inline src/Resources/Data/Users.js:8]: What is endAt here? do you mean last doc?

File: src/Resources/Reusables/Cards/Cards.jsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+const Cards = function({ items }) {
+
+  console.log(`Type of users ${typeof(items)}`);
+  
+  return (
+    <div className="cards-container">
+      {
+        items.map((item) => {
+          return (
+            <DisplayCard name={item.name} phone={item.phone} image={item.photo}/>
```
Comment (MohamedAssaf) [inline src/Resources/Reusables/Cards/Cards.jsx:11]: Love the modularity and abstraction.

File: src/Utilities/Phrases.js (hunk 1 above)
Comment (yossef-khaled) [reply src/Utilities/Phrases.js]: Ooops....

File: src/Resources/Data/Users.js (hunk 2 above)
Comment (yossef-khaled) [reply src/Resources/Data/Users.js:8]: It is a function inside the `db.Collection` instance. Where you can limit the end of your query data by typing `db.Collection.endAt(THE_LAST_DOC_OF_THE_QUERY)`

File: src/Resources/Reusables/Cards/Cards.jsx (hunk 3 above)
Comment (yossef-khaled) [reply src/Resources/Reusables/Cards/Cards.jsx:11]: Trynna catch you up.

**Comment by MohamedAssaf** [comment]
QA Comments ( open for discussion ) : 

Desktop view comments: 
1. Search icon and looks off with grey background can we remove the grey background and just have the search icon?
2. Can we place the Search label and the input and the icon on the same line? 
3. Can we place a clear results button? I had to reload to be able to view all again? 
4. There is no Arabic it's all english. ( السابق، التالي, رقم التليفون، البحث برقم التليفون )
5. Margin between Previous and next buttons
6. Pagination working more than fine.

**Comment by yossef-khaled** [comment]
On it.

**Comment by yossef-khaled** [comment]
Screenshot 2022-04-17 at 23-45-36 Salam سلام
Screenshot 2022-04-17 at 23-45-11 Salam سلام
Screenshot 2022-04-17 at 23-44-42 Salam سلام

**Pull request merged**